src/script/externalRoute.ts

```diff
@@ -90,3 +101,15 @@ export const addLocaleToUrl = (url?: string): string => {
   const websiteLanguage = language == 'de' ? language : 'en';
   return url.replace(Config.getConfig().URL.WEBSITE_BASE, `${Config.getConfig().URL.WEBSITE_BASE}/${websiteLanguage}`);
 };
+
+export const addLocaleToHelpCenterUrl = (url?: string): string => {
```

Comment:
```suggestion
const addLocaleToHelpCenterUrl = (url?: string): string => {
```

Let's keep it private for now